package.json

```diff
@@ -1,37 +1,37 @@
 {
-  "name": "@lifeomic/eslint-config-standards",
+  "name": "@lifeomic/eslint-config",
   "version": "0.0.0",
   "main": "index.js",
-  "repository": "git@github.com:lifeomic/eslint-config-standards",
+  "repository": {},
   "author": "Development <development@lifeomic.com>",
   "license": "MIT",
   "publishConfig": {
     "access": "public"
   },
   "engines": {
-    "node": ">=12"
+    "node": ">=14.17"
   },
   "scripts": {
     "test": "eslint .",
     "build": "sh ./build.sh"
   },
   "devDependencies": {
-    "conventional-changelog-conventionalcommits": "^4.6.0",
-    "eslint": "^7.32.0",
+    "conventional-changelog-conventionalcommits": "^4.6.1",
+    "eslint": "^8.0.1",
     "eslint-config-eslint": "^7.0.0",
-    "eslint-plugin-eslint-plugin": "^3.5.3",
+    "eslint-plugin-eslint-plugin": "^4.0.1",
     "eslint-plugin-node": "^11.1.0",
-    "jest": "^27.1.0",
-    "semantic-release": "^17.4.7"
+    "jest": "^27.3.1",
+    "semantic-release": "^18.0.0"
```

Comment:
Can we avoid updating this for now? Almost all of the SkillSpring ecosystem is still using Node 12.

Reply:
> I mean concerns about updating the projects to use 14

Ah, I'm not sure about this -- we haven't started looking into from SkillSpring. It just seems like this would create an unnecessary pain point -- do we need new features from `semantic-release@18` in this package?

Reply:
Also, I think this "engine" requirement isn't "technically" correct -- you'd need Node 14 to work in this repo, but not to actually consume this package (since `semantic-release` is just a devDependency)

Reply:
That is true.